frontend/apps/pablo/sentry.client.config.js

```diff
@@ -0,0 +1,20 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  environment: process.env.NODE_ENV === "production" ? "Release" : "Preview",
```

Comment:
We've got multiple preview environments right? Should we instead of Preview have the environment name that vercel assigns/the git commit hash? Or does sentry not allow for dynamic environements?